.github/workflows/docs.yml

```diff
@@ -0,0 +1,203 @@
+          echo "---" >> ${TRANSFORMED_FILE}
+          echo "" >> ${TRANSFORMED_FILE}
+          cat ${TEMP_FILE} >> ${TRANSFORMED_FILE}
+          rm ${TEMP_FILE}
+          cp ${TRANSFORMED_FILE} ${BASE_DST}/design.md
+
+          #####################################
+          #  Environment Variables            #
+          #####################################
+          TEMP_FILE="temp_setting-variables.md"
+          tail -n +2 ${BASE_SRC}/docs/Environment.md > ${TEMP_FILE}
+          TRANSFORMED_FILE="transformed/setting-variables.md"
+          echo "---" > ${TRANSFORMED_FILE}
+          echo "sidebar_label: Environment Variables" >> ${TRANSFORMED_FILE}
+          echo "sidebar_position: 230" >> ${TRANSFORMED_FILE}
+          echo "title: Setting Environment Variables" >> ${TRANSFORMED_FILE}
+          echo "tags: [rsk, rootstock, rif, flyover, integrate, integration guide, rbtc, powpeg]" >> ${TRANSFORMED_FILE}
+          echo "description: The Flyover SDK streamlines integration between client applications and the Flyover Protocol. This easy-to-use JavaScript/TypeScript toolkit provides configuration options for Liquidity Providers (LPs) and custom network setups for connecting to Rootstock." >> ${TRANSFORMED_FILE}
+          echo "---" >> ${TRANSFORMED_FILE}
+          echo "" >> ${TRANSFORMED_FILE}
+          cat ${TEMP_FILE} >> ${TRANSFORMED_FILE}
+          rm ${TEMP_FILE}
+          cp ${TRANSFORMED_FILE} ${BASE_DST}/setting-variables.md
+
+          #####################################
+          #  LP Management                    #
+          #####################################
+          TEMP_FILE="temp_management.md"
+          tail -n +2 ${BASE_SRC}/docs/LP-Management.md > ${TEMP_FILE}
+          TRANSFORMED_FILE="transformed/management.md"
+          echo "---" > ${TRANSFORMED_FILE}
+          echo "sidebar_label: LP Management" >> ${TRANSFORMED_FILE}
+          echo "sidebar_position: 220" >> ${TRANSFORMED_FILE}
+          echo "title: Liquidity Provider (LP) Management" >> ${TRANSFORMED_FILE}
+          echo "tags: [rsk, rootstock, rif, flyover, integrate, integration guide, rbtc, powpeg]" >> ${TRANSFORMED_FILE}
+          echo "description: The Flyover SDK streamlines integration between client applications and the Flyover Protocol. This easy-to-use JavaScript/TypeScript toolkit provides configuration options for Liquidity Providers (LPs) and custom network setups for connecting to Rootstock." >> ${TRANSFORMED_FILE}
+          echo "---" >> ${TRANSFORMED_FILE}
+          echo "" >> ${TRANSFORMED_FILE}
+          cat ${TEMP_FILE} >> ${TRANSFORMED_FILE}
+          rm ${TEMP_FILE}
+          cp ${TRANSFORMED_FILE} ${BASE_DST}/management.md
+
+          #####################################
+          #  Integration Tests                #
```

Comment:
I'd remove this section because this test are outdate if I recall correctly, once we update them we can add it again